src/arrayMethodJoin.js

```diff
@@ -5,7 +5,52 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```
вкажи що separator за замовчуванням = ',' і не потрібно буде робити додаткових перевірок

Reply:
Thanks